packages/@aws-cdk/aws-nodejs-aspect/lib/nodejs-aspect.ts

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable brace-style */
+import { Construct, IConstruct } from 'constructs';
+import * as cdk from 'aws-cdk-lib';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import { IAspect } from 'aws-cdk-lib/core';
+import { AwsCustomResource, Provider } from 'aws-cdk-lib/custom-resources';
+import { ReceiptRuleSet } from 'aws-cdk-lib/aws-ses';
+
+/**
+ * Runtime aspect
+ */
+abstract class RuntimeAspectsBase implements IAspect {
+
+  /**
+   * The string key for the runtime
```

Comment:
This isn't descriptive of what `targetRuntime` is. How about something like, "The runtime that the aspect will target for updates while walking the construct tree."

Reply:
Also why is this public? This seems like an implementation detail.

Reply:
yeah, these strings was to pass initial lints, have more formatted version and correct typing in latest commit.